net-mgmt/pfSense-pkg-Telegraf/files/usr/local/pkg/telegraf.inc

```diff
@@ -103,6 +103,20 @@ EOD;
 		$cfg .= "\toverwrite_template = false\n";
 		$cfg .= "\ttemplate_name = \"telegraf\"\n";
 		$cfg .= "\turls = [\"" . $telegraf_conf['elasticsearch_server'] . "\"]\n";
+        } else if ((is_array($telegraf_conf['telegraf_output']) && in_array("graphite", $telegraf_conf['telegraf_output'])) || $telegraf_conf['telegraf_output'] == "graphite") {
+		$cfg .= "[[outputs.graphite]]\n";
+		$cfg .= "\tservers = [\"" . $telegraf_conf['graphite_server'] . "\"]\n";
```

Comment:
Would this not still be a problem if they failed to enter a server name? Or would graphite ignore that since it would be empty (`[""]`)? If graphite will ignore the empty server rather than fail, it may be OK as-is.

Reply:
I'm not deep into php development. I've aligned my proposal to the existing options.
It will behave the same as an empty inluxdb- server or an empty elastic-server server.

Reply:
@jim-p Hi Jim,

I helped out with the v1.6.1 Telegraf release recently #517.
Telegraf will ignore `[[outputs]]` stanza configuration keys with no value (no errors/no bailing).

@thomasDOTde's changes work fine on my dev. instance, both with a Graphite host defined and not undefined.

Reply:
So I thin this can be considered resolved? Is there anything else left to do?